.github/workflows/codeql-analysis.yml

```diff
@@ -19,24 +19,18 @@ jobs:
       security-events: write
 
     strategy:
-      fail-fast: false
       matrix:
         language: [ 'javascript' ]
```

Comment:
We don't actually need the `matrix:` here, this can just move to line 33, but when I did that in https://github.com/dependabot/updater-action/pull/76/commits/3cf6580c9c1eb57bc6803bcc12d0fb435ffa3a4b, it resulted in our continuity of CodeQL Analysis being reset as it essentially saw this as a brand new analysis.

I decided to leave it as it was to avoid any confusion.